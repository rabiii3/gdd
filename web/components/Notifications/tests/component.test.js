import React from 'react';
import { shallow } from 'enzyme';
import { Component } from '..';

describe('web | components | People | index ', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const props = {
        classes: {},
        notifications: [],
        onClose: jest.fn(),
        getEntity: jest.fn(),
        people: {}
      };
      const wrapper = shallow(<Component {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
