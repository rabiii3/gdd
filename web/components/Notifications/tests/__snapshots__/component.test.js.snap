// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`web | components | People | index  render should match snapshot 1`] = `
<WithStyles(Grid)>
  <WithStyles(MenuItem)>
    <WithStyles(Typography)
      variant="h6"
    >
      <FormattedMessage
        defaultMessage="Notifications"
        id="components.Notifications.notifications"
        values={Object {}}
      />
    </WithStyles(Typography)>
    <WithStyles(IconButton)
      onClick={[MockFunction]}
    >
      <pure(ChevronRightIcon) />
    </WithStyles(IconButton)>
    <Spacer
      size="LARGE"
    />
    <WithStyles(Grid)
      alignItems="center"
      container={true}
      direction="row"
      justify="flex-end"
    >
      <WithStyles(Button)
        onClick={[Function]}
      >
        <WithStyles(Typography)
          align="right"
          color="primary"
          variant="caption"
        >
          <FormattedMessage
            defaultMessage="Clear"
            id="components.Notifications.remove"
            values={Object {}}
          />
        </WithStyles(Typography)>
      </WithStyles(Button)>
    </WithStyles(Grid)>
  </WithStyles(MenuItem)>
  <WithStyles(Divider) />
</WithStyles(Grid)>
`;
